src/i18n/en.json

```diff
@@ -290,7 +333,7 @@
         "serialNumber": "Serial number",
         "vehicleClass": "Vehicle class",
         "vehicleInfo": "Vehicle information",
-        "vehicleCopyright": "Source: Transport register, Traficom",
+        "vehicleCopyright": "Vehicle information - Transport register, Traficom",
```

Comment:
Why this is changed now backwards? Shouldn't it be `Source: Transport register, Traficom` always?